### ISSUE TITLE
Player.Open Throwing "Invalid parameters" error.

### ISSUE THREAD

**Issue by alanjames1987** [PaulAvery/node-xbmc-ws#1]
When I tried to play a movie with a specific id I get an error. Below is the code I used to attempt to play the movie.

```JavaScript
connection.run('Player.Open')({
	'item' : {
		"movieid" : 12
	},
	'options' : {
		'resume' : true
	}
}, function(err, results) {
	console.log(err);
	console.log(results);
});
```

Adding a movieid property to the item object in the parameters has always been the way I have played movies in other XBMC libraries and as far as I can tell it is the way to do it. However, it generates this error

```
[ERROR] 12:15:33 Error
Error: Invalid parameters
```

Is this user error or a bug?

**Comment by PaulAvery** [comment]
Welp, after 1 1/2 years I finally revisited this.
I rewrote it and its underlying library to completely be based on promises.
I also removed client side validation.
So hopefully your use case should work now, albeit with the slightly different syntax of

```js
connection.Player.Open({
    'item' : {
        "movieid" : 12
    },
    'options' : {
        'resume' : true
    }
}).then(function(err, results) {
    console.log(results);
}, function(err) {
    console.log(err);
});
```

I kinda guess you gave up on this a long time ago,
I just thought I should commnent and close this issue anyways.
